tools/golangci-lint.sh

```diff
@@ -6,9 +6,9 @@ BUILD_ROOT="${REPO_ROOT}/build"
 BUILD_BIN="${BUILD_ROOT}/bin"
 
 NAME=golangci-lint
-RELEASE=v1.42.1
-OSX_RELEASE_256=9c0042e91218dc1dd4eb7b54e29c7331eff081b3ac3f88b0d5df89b976fcd45c
-LINUX_RELEASE_256=214b093c15863430c4b66dd39df677dab6e38fc873ded147e331740d50eea51f
+RELEASE=v1.47.3
+OSX_RELEASE_256=75ff9491f1ac88ab8594d0498d2a86168ad20e3f4b958dd21be8a6c2988d50cf
+LINUX_RELEASE_256=67b5f7a2275d94034c5c87ff7655ba4b20bdb5f83c43c5d22476f2bf5c6766ef
 
 ARCH=amd64
```

Comment:
curious if anyone ever runs this locally or only in CI? I'd be happy to implement the required logic to download the arm64 tarball when running on Apple Silicon.